test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -250,14 +255,16 @@ public void createParticipant() {
         STUDY.setEmailVerificationEnabled(true);
         mockHealthCodeAndAccountRetrieval();
         
+        when(externalIdService.getExternalId(STUDY.getStudyIdentifier(), EXTERNAL_ID))
+                .thenReturn(new DynamoExternalIdentifier());
```

Comment:
ExternalIdentifier.create() to avoid leaking DDB implementation to non-DDB classes. (Here and several other places in this PR.)

Reply:
Changed these (all new DynamoStudy() --> Study.create()).